cmd/opentelemetry-collector/app/exporter/grpc/factory.go

```diff
@@ -0,0 +1,87 @@
+// Copyright (c) 2020 The Jaeger Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package grpc
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/open-telemetry/opentelemetry-collector/component"
+	"github.com/open-telemetry/opentelemetry-collector/config/configerror"
+	"github.com/open-telemetry/opentelemetry-collector/config/configmodels"
+
+	storageGrpc "github.com/jaegertracing/jaeger/plugin/storage/grpc"
+)
+
+const (
+	TypeStr = "jaeger_grpc"
```

Comment:
I think this needs to be `jaeger_grpc_plugin` to avoid confusion with `jaeger grpc` exporter (used for communicating with another collector configured with jaeger grpc receiver).